src/main/resources/csv/mainnet.csv

```diff
@@ -0,0 +1,3 @@
+id|name|standard|account_id
+1 | USDTez | FA1.2 | tz1eEnQhbwf6trb8Q8mPb2RaPkNk2rN7BKi9
+
```

Comment:
I swear I'm having a deja-vu but I'll ask the same: why are we choosing csv over adding some more json config?

Reply:
not sure, in the ticket there is mentioned specific format of CSV, so I would assume that there is an easy way to extract data in it

Reply:
We can definitely switch over to JSON subsequently if it makes more sense. The choice of CSV in the spec was pretty arbitrary.

Reply:
let's put this in a new issue?